src/Microsoft.Azure.WebJobs.Extensions.Kafka/Config/KafkaOptions.cs

```diff
@@ -136,6 +136,14 @@ public int MaxBatchSize
         /// <value>The auto commit interval ms.</value>
         public int AutoCommitIntervalMs { get; set; } = 200;
 
+        /// <summary>
+        /// Gets or sets the debug option for librdkafka library.
+        /// Default = "" (disable)
+        /// 
+        /// Librdkafka: debug
+        /// </summary>
+        public string LibkafkaDebug { get; set; } = null; 
```

Comment:
We could add the possible values here:
"A comma-separated list of debug contexts to enable: all,generic,broker,topic,metadata,producer,queue,msg,protocol,cgrp,security,fetch"

Reply:
Done